alignak/scheduler.py

```diff
@@ -119,16 +119,15 @@ def __init__(self, scheduler_daemon):
         self.recurrent_works = {
             0: ('update_downtimes_and_comments', self.update_downtimes_and_comments, 1),
             1: ('schedule', self.schedule, 1),  # just schedule
-            2: ('consume_results', self.consume_results, 1),  # incorporate checks and dependencies
+            2: ('check_freshness', self.check_freshness, 10),
```

Comment:
The poller? but freshness is for passive, so we create check result, so we have the result immediatly, it's more logical like this :p